dsaas-services/rh-che.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+- hash: 808bc0e456437ab757cda1f23e43c06cac694b4b
+  hash_length: 7
+  name: che
+  parameters:
+    IMAGE: push.registry.devshift.net/che/che-multiuser
```

Comment:
Remove `push.` from registry URL and it should work

Reply:
thanks, lemme try